client/src/types.ts

```diff
@@ -1,3 +1 @@
-import {CSSProperties} from 'react';
-
-export type Styles = {[name: string]: CSSProperties};
+export type Styles = {[name: string]: string};
```

Comment:
hmm there might be a type for this in Emotion's `index.d.ts`

Reply:
There is a type of classNameArg from emotion that seems like it would fit, but when using I'm getting conflicting type error with className. Using type string works fine though.

Reply:
👍🏽 should be good for now